lib/stylelint/processorRemoveMixins.js

```diff
@@ -0,0 +1,7 @@
+// If this proposed functionality ever gets added https://github.com/styled-components/stylelint-processor-styled-components/issues/271
+// then please delete this processor and any use of it.
+
+module.exports = () => ({
+  code: input =>
+    input.replace(/-styled-mixin\d+:\s.+(?!{|,).$/gm, `/* styled-mixin */`),
```

Comment:
Was there any particular reason/preference why we are commenting out rather than removing the styled-mixin?

Reply:
Yeh there is. By removing the line with the styled-mixin it can open a can of worms of linting errors such as having a selector with no styles e.g.

```jsx
const example = css`
  ${({ height }) => height ? `height: ${height};` : ''}
`;
```
would produce

```jsx
.selector1 {
}
```
and also having a semi-colon where you don't need one e.g.

```jsx
const example = css`
  width: 100%;
  ${({ height }) => height ? `height: ${height};` : ''}
`;
```

 would produce

```jsx
.selector1 {
  width: 100%; <--- semi colon not needed
}
```

Reply:
The empty selector makes sense, that's not ideal. On the semi-colon is that suggesting the semi-colon's are only needed in CSS if there is a trailing statement?

Reply:
Yes, that's true but after investigation I realised it's bad practice to do that and the semi-colon error must have been related to something else.